packages/ember-flight-icons/package.json

```diff
@@ -33,24 +33,26 @@
     "ember-get-config": "^2.1.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.24.5",
+    "@babel/core": "^7.24.7",
     "@babel/eslint-parser": "^7.22.15",
     "@babel/plugin-proposal-decorators": "^7.23.7",
     "@babel/plugin-transform-class-properties": "^7.23.3",
-    "@ember/optional-features": "^2.0.0",
+    "@babel/preset-typescript": "^7.24.7",
+    "@ember/optional-features": "^2.1.0",
     "@embroider/addon-dev": "^4.3.1",
     "@embroider/test-setup": "^3.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "@rollup/plugin-babel": "^6.0.4",
+    "concurrently": "^8.2.2",
     "ember-template-lint": "^4.18.2",
     "ember-template-lint-plugin-prettier": "^4.2.0",
-    "eslint": "^8.52.0",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-ember": "^11.11.1",
+    "eslint": "^8.57.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-ember": "^11.12.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-qunit": "^8.0.1",
+    "eslint-plugin-qunit": "^8.1.1",
     "npm-run-all": "^4.1.5",
```

Comment:
same here, regarding `npm-run-all`

Reply:
Good catch. I removed it and added that to to the initial commit.